spec/money_spec.rb

```diff
@@ -330,6 +330,13 @@
 
   end
 
+  describe "value" do
+    it "rounds to the number of minor units provided by the currency" do
+      expect(Money.new(1.1111, 'USD').value).to eq(1.11)
```

Comment:
Nitpick but for completeness sake lets add a currency like TND in here too which should result in 1.111.